libr/core/cmd_write.c

```diff
@@ -94,21 +94,22 @@ static void cmd_write_op (RCore *core, const char *input) {
 	ut8 *buf;
 	int len;
 	const char* help_msg[] = {
-		"Usage:","wo[asmdxoArl24]"," [hexpairs] @ addr[!bsize]",
+		"Usage:","wo[asmdxoArl24][y]"," [hexpairs] @ addr[!bsize]",
 		"wow"," [val]", "==  write looped value (alias for 'wb')",
 		"woa"," [val]", "+=  addition (f.ex: woa 0102)",
 		"wos"," [val]", "-=  substraction",
 		"wom"," [val]", "*=  multiply",
 		"wod"," [val]", "/=  divide",
-		"woe"," [from to] [step] [wsz=1]","..  create sequence",
 		"wox"," [val]","^=  xor  (f.ex: wox 0x90)",
 		"woo"," [val]","|=  or",
 		"woA"," [val]","&=  and",
-		"woR","","random bytes (alias for 'wr $b')",
 		"wor"," [val]", ">>= shift right",
 		"wol"," [val]","<<= shift left",
 		"wo2"," [val]","2=  2 byte endian swap",
 		"wo4"," [val]", "4=  4 byte endian swap",
+		"wo*y", "", "same as their counterparts above, but using clipboard instead of [val]",
```

Comment:
Into defense? Dont take me wrong. Im not trying to be offensive or anything. Just trying to give some feedback on every new contributor just to discuss better way to handle this.

Thats a very nice addition :) take a beer and chill a bit ;)

Ill merge after lunch

Thx

> On 07 Apr 2016, at 14:49, Christiane Ruetten <notifications@github.com> wrote:
> 
> In libr/core/cmd_write.c:
> 
> >  		"wor"," [val]", ">>= shift right",
> >  		"wol"," [val]","<<= shift left",
> >  		"wo2"," [val]","2=  2 byte endian swap",
> >  		"wo4"," [val]", "4=  4 byte endian swap",
> > +		"wo*y", "", "same as their counterparts above, but using clipboard instead of [val]",
> Thanks for the explanation. I'm sorry. I didn't mean to drive you into defense. :/
> 
> I will push along a separate commit in this pull request.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>